copter/source/docs/tuning-process-instructions.rst

```diff
@@ -9,26 +9,138 @@ Setting the aircraft up ready for tuning
 
 The following parameters should be set correctly based on the specifications of your aircraft.
 
-- MOT_BAT_VOLT_MAX - 4.2 x No. Cells
-- MOT_BAT_VOLT_MIN - 3.3 x No. Cells
-- MOT_PWM_MAX - Check ESC manual for fixed range or 2000
-- MOT_PWM_MIN - Check ESC manual for fixed range or 1000
-- MOT_SPIN_ARM - use INITIAL SETUP -> OPTIONAL HARDWARE -> MOTOR TEST facility in mission planner to check.
-- MOT_SPIN_MAX - 0.95
-- MOT_SPIN_MIN - use INITIAL SETUP -> OPTIONAL HARDWARE -> MOTOR TEST facility in mission planner to check.
-- MOT_THST_EXPO - 0.55 for 5 inch props, 0.65 for 10 inch props, 0.75 for 20 inch props. This parameter should be derived by thrust stand measurements for best results (don’t trust manufacture data).
-- MOT_THST_HOVER - 0.25 or below the expected hover thrust percentage (low is safe)
-- INS_ACCEL_FILTER -  10 to 20
-- INS_GYRO_FILTER - 80 for 5 inch props, 40 for 10 inch props, 20 for 20 inch props
-- ATC_RAT_PIT_FILT - INS_GYRO_FILTER / 2
-- ATC_RAT_RLL_FILT - INS_GYRO_FILTER / 2
-- ATC_RAT_YAW_FILT - 2
-- ATC_ACCEL_P_MAX - 110000 for 10 inch props, 50000 for 20 inch props, 20000 for 30 inch props
-- ATC_ACCEL_R_MAX - 110000 for 10 inch props, 50000 for 20 inch props, 20000 for 30 inch props
-- ATC_ACCEL_Y_MAX - 27000 for 10 inch props, 18000 for 20 inch props, 9000 for 30 inch props 
-- ACRO_YAW_P - 0.5 x ATC_ACCEL_Y_MAX / 4500
-
-The initial tune of the aircraft should be done in the aircrafts most agile configuration. This generally means that the aircraft will be at its minimum take off weight with fully charged batteries.
+.. warning::
+
+    Values provide here are a starting point to tune your aircraft, they should not be hardly trust as the best values. Before flying please read the entire instruction to understand what you will do, what you should expect and what to do.
+
+Battery setting
+^^^^^^^^^^^^^^^
+Those values a necessary to scale the motor thrust linearisation.
+
+- :ref:`MOT_BAT_VOLT_MAX <MOT_BAT_VOLT_MAX>` - 4.2v x No. Cells
+- :ref:`MOT_BAT_VOLT_MIN <MOT_BAT_VOLT_MIN>` - 3.3v x No. Cells
+
+Motors setup
+^^^^^^^^^^^^
+Those values are necessary to scale correctly your motor control.
```

Comment:
Parameters used to define the output range sent to the ESC.

This is probably closer but not perfect.